bin/alluxio-mount.sh

```diff
@@ -20,7 +20,7 @@ if [[ "$-" == *x* ]]; then
 fi
 BIN=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
 
-USAGE="Usage: alluxio-mount.sh [Mount|SudoMount] [MACHINE]
+USAGE="Usage: alluxio-mount.sh --umount [Mount|SudoMount|Umount|SudoUmount] [MACHINE]
```

Comment:
Do we need `--umount`? If we do, do we need to update alluxio-start?

Reply:
sorry, the `--umount` was added accidentally. Removed it now